### PR TITLE
Add deploy configuration

### PR DESCRIPTION
**Pull request by voyager-gold**
Add configuration to deploy on BSC testnet
https://github.com/CoMakery/comakery-security-token/issues/39

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -23,13 +23,35 @@ task("accounts", "Prints the list of accounts", async () => {
...
+      },
+    },
+  },
+  namedAccounts: {
+    deployer: 0,
+  },
   networks: {
-    hardhat: {
-      allowUnlimitedContractSize: true
+    testnet: {
```
Comment (aquabu) [inline hardhat.config.js:43]: We're going to need to be able to deploy to a number of different networks. How would we switch networks?

Comment (aquabu) [reply hardhat.config.js:43]: This may just need clarification in the README about how to configure and deploy to different networks.

Comment (voyager-gold) [reply hardhat.config.js:43]: `npx hardhat deploy --network <network defined in hardhat.config.js>`

Comment (voyager-gold) [reply hardhat.config.js:43]: Okay I will put description in readme. Thanks for give me the tip!

**Review by aquabu — APPROVED** [review]

**Pull request merged**